### ISSUE TITLE
Benchmarks against CPP Python Java similar data structure

### ISSUE THREAD

**Issue by nikolaydubina** [elliotchance/orderedmap#18]
It would be illustrative to compare against other main languages, particularly CPP so that people see if this is a good fit for their project!

**Comment by soypat** [comment]
Yes! And also would not hurt to include a benchmark against the intrinsic Go native map.
